client/src/partials/patients/registry/search.modal.html

```diff
@@ -5,17 +5,17 @@
   bh-form-defaults
   novalidate>
 
-  <div class="modal-header">
+  <div class="modal-header" style="box-shadow : 0px 2px 4px -4px black; z-index:1000">
     <ol class="headercrumb">
       <li class="static">{{ "FORM.INFO.PATIENTS" | translate }}</li>
       <li class="title">{{ "FORM.INFO.SEARCH" | translate }}</li>
     </ol>
   </div>
 
-  <div class="modal-body modal-tall" style="overflow: auto;">
+  <div class="modal-body" style="max-height : 400px; overflow: auto;">
```

Comment:
One resizes with the screen and the other doesn't - I think in this case it would be better for the modal to not stretch. 

I agree this should be standardized though - there could be two classes if both of these behaviours are required.